src/main/java/picard/analysis/CollectWgsMetricsWithNonZeroCoverage.java

```diff
@@ -54,7 +54,7 @@
 )
 public class CollectWgsMetricsWithNonZeroCoverage extends CollectWgsMetrics {
 
-    static final String USAGE_SUMMARY = "(Experimental) Collect metrics about coverage and performance of whole genome sequencing (WGS) experiments.  ";
+    static final String USAGE_SUMMARY = "Collect metrics about coverage and performance of whole genome sequencing (WGS) experiments.  ";
```

Comment:
For these tools where the text is removed, the corresponding annotation should be added.

Reply:
There is already used corresponding annotation `@BetaFeature`
![screenshot from 2018-01-31 10-25-03](https://user-images.githubusercontent.com/18378140/35610191-6b9f3a68-0671-11e8-8d0e-644922539b53.png)

Reply:
hmmm. I think that this should be experimental @nh13 how useful have you been finding this tool to be? Can you elaborate on the type of experiments for which you'd recommend using it?